src/index.js

```diff
@@ -41,8 +59,46 @@ const createTask = () => { // eslint-disable-line no-unused-vars
     li.appendChild(div);
     ul.appendChild(li);
     output.appendChild(ul);
+
+    // Event to change dots icon to trash icon after double clicking.
+    verticalDots.addEventListener('dblclick', () => {
+      const trash = document.createElement('i');
+      trash.classList.add('bi', 'bi-trash');
+      trash.style.cursor = 'pointer';
+      verticalDots.remove();
+      div.append(label, trash);
```

Comment:
- [OPTIONAL] This event listener works fine. The issue is that it will be hard for the user to figure out they are supposed to double click for the delete icon to appear. May just use click OR add an instruction on the user interface. I suggest making it a click event, that is more affordable.